plugins/database/postgresql/postgresql_test.go

```diff
@@ -1268,139 +1536,6 @@ func TestNewUser_CloudGCP(t *testing.T) {
 	}
 }
 
-// This is a long-running integration test which tests the functionality of Postgres's multi-host
-// connection strings. It uses two Postgres containers preconfigured with Replication Manager
-// provided by Bitnami. This test currently does not run in CI and must be run manually. This is
-// due to the test length, as it requires multiple sleep calls to ensure cluster setup and
-// primary node failover occurs before the test steps continue.
-//
-// To run the test, set the environment variable POSTGRES_MULTIHOST_NET to the value of
-// a docker network you've preconfigured, e.g.
-// 'docker network create -d bridge postgres-repmgr'
-// 'export POSTGRES_MULTIHOST_NET=postgres-repmgr'
-func TestPostgreSQL_Repmgr(t *testing.T) {
```

Comment:
This doesn't work for me as I think the image was removed from the docker mirror registry? It has been replaced by `TestPostgreSQL_InitializeMultiHost` added in this PR which is more lightweight and runs in CI.